### PR TITLE
Draft Pull-request for current progress

### PR DESCRIPTION
**Pull request by gbhandarwar**
Current progress

### CODE REVIEW

**Comment by vynpt** [comment]
removed env and cache files.

File: demoapp/app.py [hunk 1]
```diff
@@ -1,149 +1,247 @@
...
+template = """"You are a summarizer model that summarizes legal bills and legislation. Please include the bill's main purpose, relevant key points and any amendements. 
+The summaries must be easy to understand and accurate based on the provided bill. I want you to summarize the legal bill and legislation. 
+Use the title {title} to guide your summary. Summarize the bill that reads as follows:\n{context}\n\nSummary: An Act [bill title]. This bill [key information].
+"""
+
+# model to test hallucination
+# model = CrossEncoder('vectara/hallucination_evaluation_model')
+
+# load the dataset
+df = pd.read_csv("demoapp/all_bills.csv")
```
Comment (funkyvoong) [inline demoapp/app.py:42]: This file is missing from the demoapp folder, please include a a link to the file or upload it

**Review by weibb123 — APPROVED** [review]
most up-to-date changes

File: demoapp/app.py (hunk 1 above)
Comment (weibb123) [reply demoapp/app.py:42]: Hi Michelle, the Vectara/hallucination_evaluation_model is to call the model_path url to retrieve the information. We didn't store the model locally.
https://huggingface.co/vectara/hallucination_evaluation_model

Comment (funkyvoong) [reply demoapp/app.py:42]: Apologies for the confusion, I was referring to line 42

Comment (weibb123) [reply demoapp/app.py:42]: Hmm, I saw this file exist in the dev branch. Perhaps it is included from previous request?

pic1

Comment (vynpt) [reply demoapp/app.py:42]: yeah all_bills.csv is already in BU Spark's dev branch

**Pull request merged**